README.rst

```diff
@@ -160,6 +160,15 @@ Please read our
 `coafile specification <http://docs.coala.io/en/latest/Users/coafile.html>`__
 to learn more.
 
+Using ``default_coafile``
+*************************
+
+* The content of ``default_coafile`` is similar to that of ``.coafile``,
+  but it is system wide.
```

Comment:
I think this can be improved like `default_coafile is a normal coala Configuration file with system wide effect.`
Also, we can state that the `default_coafile` is saved in coala installation directory.